templates/openstackclient/bin/init.sh

```diff
@@ -17,5 +17,20 @@ set -ex
 
 # if the pvc is an empty volume, copy the existing hosts file to it
 if [ ! -f /mnt/hosts ]; then
-  cp /etc/hosts /mnt
+  cp /etc/hosts /mnt/hosts/
```

Comment:
Should this maybe be `/mnt/hosts`, i.e. leave off the trailing `/`?  Just checking

Reply:
no, I changed to mount the volume where we at the moment store the permanent /etc/hosts file from /mnt to /mnt/hosts, maybe /mnt/etc is a better choice

Reply:
changed this to /mnt/etc to be more clear